### PR TITLE
Fix broken tests

### PR DESCRIPTION
**Pull request by Qard**
This fixes a few minor problems with the test suite.

### CODE REVIEW

**Comment by Qard** [comment]
The final failing test appears to be an issue with async-listener, so I'll leave this open for now while I look into that.

**Comment by DeTeam** [comment]
@Qard any success with async-listener investigation?

File: context.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ if (!process.addAsyncListener) require('async-listener');
 function Namespace(name) {
   this.name   = name;
   // changed in 2.7: no default context
-  this.active = null;
+  this.active = Object.create(null);
```
Comment (DeTeam) [inline context.js]: Why?

**Comment by othiym23** [comment]
Landed as 33d2529cadda8661d0310380d1a951f2677cae1f, although I might further tweak to remove

```diff
diff --git a/context.js b/context.js
index 7eb1cab..e30bb80 100644
--- a/context.js
+++ b/context.js
@@ -17,7 +17,7 @@ if (!process.addAsyncListener) require('async-listener');
 function Namespace(name) {
   this.name   = name;
   // changed in 2.7: no default context
-  this.active = null;
+  this.active = Object.create(null);
   this._set   = [];
   this.id     = null;
 }
```

@Qard, do you remember why this change was necessary? The comment right above that line seems to contradict the change.

**Comment by Qard** [comment]
To be honest, I don't entirely remember. I seem to recall something about some weirdness due to the `this._set.push(this.active)` in `enter()` pushing nulls into the list, but I don't remember why that was an issue. Looking at it now, I don't see any reason it can't just be null, assuming the tests pass.

**Comment by othiym23** [comment]
Reverted in cfe82da. Thanks for the explanation, @Qard!

**Pull request closed without merging**